lwt/cohttp_lwt_unix_net.ml

```diff
@@ -43,15 +43,10 @@ let connect_uri ~ctx uri =
   >>= fun client ->
   Conduit_lwt_unix.connect ~ctx:ctx.ctx client
 
-let close_in ic =
-  ignore_result (try_lwt Lwt_io.close ic with _ -> return_unit)
+let close c = Lwt.catch (fun () -> Lwt_io.close c) (fun _ -> return_unit)
```

Comment:
It seems like `close'` was just a helper that was never exposed. It's no longer necessary.